src/main/resources/Diagram/DiagramEditSheet.xml

```diff
@@ -773,7 +773,7 @@ define('diagramStore', ['jquery', 'xwiki-meta', 'xwiki-utils', 'draw.io', 'xwiki
       }
       // Append source parameter to the query
       queryString += '&amp;source=url';
-      return window.location.protocol + window.location.host +
+      return window.location.protocol + '//' + window.location.host +
```

Comment:
It's a bit unexpected that the protocol includes the ``:`` but that is the case.. https://developer.mozilla.org/en-US/docs/Web/API/HTMLHyperlinkElementUtils/protocol